### PR TITLE
oma: Import from brewsci/science

### PR DESCRIPTION
**Pull request by alpae**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

**Comment by sjackman** [comment]
Please remove the merge commit from this PR
917dce9 `Merge branch 'master' into migrate-oma`

File: Formula/oma.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Oma < Formula
+  desc "Standalone package to infer orthologs with the OMA algorithm"
+  homepage "https://omabrowser.org/standalone/"
+  url "https://omabrowser.org/standalone/OMA.2.2.0.tgz"
+  sha256 "25ee17b92ef6631507311a93036c91154a42bcee8d033e3a3912836f2764040d"
+
+  depends_on "python" => :recommended
```
Comment (sjackman) [inline Formula/oma.rb]: Please remove `=> :recommended`. It's better to avoid options when possible.

File: Formula/oma.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+class Oma < Formula
+  desc "Standalone package to infer orthologs with the OMA algorithm"
+  homepage "https://omabrowser.org/standalone/"
+  url "https://omabrowser.org/standalone/OMA.2.2.0.tgz"
+  sha256 "25ee17b92ef6631507311a93036c91154a42bcee8d033e3a3912836f2764040d"
```
Comment (sjackman) [inline Formula/oma.rb]: If OMA is published, please add
```ruby
# cite "http://doi.org/…"
```

Comment (sjackman) [reply Formula/oma.rb]: For example…
https://github.com/brewsci/homebrew-bio/blob/aec1199245fb8e47d65f2b5205d78367cb39c8e5/Formula/adam.rb#L6

**Comment by alpae** [comment]
Hi @sjackman thanks for the comments. I changed accordingly and force pushed. I don't understand what is happening with the build-macos circleCI tests. I don't think it is related to the OMA formula, is it? Thanks for your help and the effort to keep a science tap for homebrew!

**Comment by sjackman** [comment]
```
==> brew fetch --retry makedepend pkg-config
==> FAILED
Fetching: makedepend, pkg-config
==> Downloading https://linuxbrew.bintray.com/bottles/makedepend-1.0.5.sierra.bottle.1.tar.gz
curl: (22) The requested URL returned error: 404 Not Found
```
https://circleci.com/gh/brewsci/homebrew-bio/212
Yep, that error is definitely not caused by your PR. I'm hoping that PR https://github.com/brewsci/homebrew-bio/pull/21 will resolve this issue.

**Comment by sjackman** [comment]
I've merged PR #21 and triggered a rebuild of your PR on CircleCI for macOS. Fingers crossed…

**Comment by sjackman** [comment]
Rebuilding wasn't sufficient. I rebased your PR on top of origin/master.

**Comment by sjackman** [comment]
Merged. Thanks for your contribution to Brewsci/bio, Adrian!

**Pull request closed without merging**